elastic/beats
# Osquerybeat: Disable allow_unsafe osquery configuration

## Proposed commit message

Osquerybeat: Disable ```allow_unsafe``` osquery configuration

The ```allow_unsafe``` flag was used for windows only since Oct 21, 2021.

This addresses the issue reported in:
https://github.com/elastic/security/issues/2194

## Checklist


## Disruptive User Impact

If any of the existing users used their custom extensions since 2021 without proper permissions set (the osquerybeat was careful to preserve modifications to the autoload file in order to allow these use cases)
https://osquery.readthedocs.io/en/stable/deployment/extensions/#extensions-binary-permissions

then these extensions will fail to load.

## How to test this PR locally

Install the Agent with osquery manager integration in the policy. Make sure the extension loads.

## Related issues

- Closes https://github.com/elastic/security/issues/2194
